generateIndices.py

```diff
@@ -0,0 +1,81 @@
+#reads in files from features.npz
+# output training and testing indices
+import sys
+import re
+import math
+import numpy as np
+
+def generateIndex():
+
+    feature_file = sys.argv[1]
+
+    features_files = np.load(feature_file)
+    filenames = features_files['filenames']
+
+    #master list to hold information
+    master = []
+    #number of files in each pile
+    numberOfFiles_0=0
+    numberOfFiles_1=0
+    numberOfFiles_2=0
+    numberOfFiles_3=0
```

Comment:
A hash or list is better than 4 named variables. (so you don't have to repeat lines each step for each variable, you just loop through them)